docs/conf.py

```diff
@@ -268,4 +268,7 @@ def setup(app):
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
+intersphinx_mapping = {
+    'http://docs.python.org/': None,
```

Comment:
This lets use cross-reference to things defined in the official Python documentation, although it uses the old `intersphinx_mapping` format, and I don't think we're using it at all. I'll see if it can be removed without problems, Sphinx prints compilation warnings for any broken refs so it's easy to find out if we have any refs that use this...